Cargo.toml

```
@@ -37,7 +37,7 @@ complex = ["dep:num-complex"]
 
 [dependencies]
 conv = "0.3.3"
-libffi = { version = "3.2.0", optional = true }
+libffi = { version = "4.0.0", optional = true }
```

```suggestion
libffi = { version = "4.1.0", optional = true }
```
I see a new version is out. And please update the `rust-version` field; then we should be able to merge this.